framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1680,6 +1680,12 @@ public final String getOption(String name) {
     // TODO I would like to rename getLintOption to hasLintOption
     @Override
     public final boolean hasOption(String name) {
+
+        // Always ignore errors from raw type arguments.
+        // See https://github.com/typetools/checker-framework/issues/635
+        if (name.equals("ignoreRawTypeArguments")) {
```

Comment:
Change

```
if (hasOption("ignoreRawTypeArguments") ...
```

into

```
if (getOption("ignoreRawTypeArgument", "true").equals("true")) ...
```

Then when somebody wants the stricter checks, they can pass `-AignoreRawTypeArguments=false`.